src/box/lua/upgrade.lua

```diff
@@ -1241,6 +1241,28 @@ local function upgrade_to_2_11_0()
 end
 --------------------------------------------------------------------------------
 
+--------------------------------------------------------------------------------
+-- Tarantool 2.11.1
```

Comment:
I think we usually use the name of the current version for the upgrade, not the next one. I mean, since the `2.11.1` tag is still not set, and `2.11.0-entrypoint` already exists, it might be better to move this function to the `2.11.0` schema version. Although I may be wrong.

Reply:
It seems, it should be backported to 2.10, so, maybe, 2.11.1 is not the best name (it seems, 2.10.4 fits better). OTOH, if existing 2.11.0 migration already applied, 2.10.4 will not be applied (see #4804). It shouldn't affect upgrades from a release to a release, but may hit developers a bit.

Reply:
I think in the case of 2.10 it just needs to be properly backported. I mean it should be done in another PR.

Reply:
Good idea (2.11.0 in 2.11 and 2.10.4 in 2.10).

Regarding adding to existing 2.11.0 upgrade: it follows our practice. I don't like it (I would link #4804 here again), but we can't do more without refactoring. Or really disregard 3rd version number connection beetween tarantool runtime and tarantool schema and name the new upgrade as 2.11.1.

Reply:
Please check the updated version and verify that I understood you correctly.
Note that the last commit can be simply backported to 2.10.

Reply:
I agree that this will work, although I'm a little uncomfortable creating an update to `2.10.4` after `2.11.0-entrypoint`. Also, I think to be able to backport to 2.10 you need to remove `revoke_execute_access_to_lua_function_from_role_public()` from `upgrade_to_2_11_0()` as suggested by @locker.

Reply:
Thanks, removed.

Reply:
As Vladimir stated below, there were no 2.11 releases, so it'll work. It is technically correct, so I technically agree. You know my opinion on given upgrade approach in general.